source/tank/Tank.hx

```diff
@@ -25,6 +26,7 @@ class Tank extends FlxSprite {
 		initSelf(bodyGraphic);
 		initCannon();
 		initBullets();
+		this.elasticity = 1;
```

Comment:
this is a pretty interesting change. I don't mind it being applied to all tanks, but right now it seems like it only has a practical impact on our DynamicTank - maybe it's appropriate to just make our factory method do this line for Dynamic tanks and leave default tank behavior unchanged?

Reply:
Sounds good, yeah I noticed it didn't do anything to any other types of tanks. I decided to add the elasticity into the DynamicMovement controller and keep TankFactory as is.